master/static/migrations/20240520133440_add-arbitrary-metadata-table.tx.up.sql

```diff
@@ -0,0 +1,21 @@
+-- CREATE RUN METADATA TABLE
+CREATE TABLE runs_metadata (
+    run_id INTEGER PRIMARY KEY,
+    metadata JSONB,
+    FOREIGN KEY (run_id) REFERENCES runs(id) ON DELETE CASCADE
+);
+
+-- CREATE INDEX TABLE
+CREATE TABLE runs_metadata_index (
+    id SERIAL PRIMARY KEY,
```

Comment:
```suggestion
    id integer PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
```

https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_serial